client/blocks/app-banner/docs/example.jsx

```diff
@@ -0,0 +1,34 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import AppBanner from '../';
+import { PREFERENCE_NAME } from '../utils';
+import { savePreference } from 'state/preferences/actions';
+
+function AppBannerExample( clearPreference ) {
```

Comment:
If we have trouble adding this as an example in devdocs, feel free to remove this file.